src/connector.ts

```diff
@@ -371,15 +370,23 @@ function createCLJSReplType(
         'cljsReplTypeHasBuilds',
         cljsType.buildsRequired
       );
-      let initCode = cljsType.connectCode,
-        build: string = null;
+      let initCode: typeof cljsType.connectCode | undefined = cljsType.connectCode,
+        build: string | null = null;
       if (menuSelections && menuSelections.cljsDefaultBuild && useDefaultBuild) {
         build = menuSelections.cljsDefaultBuild;
         useDefaultBuild = false;
       } else {
         if (typeof initCode === 'object' || initCode.includes('%BUILD%')) {
+          const buildsForSelection = startedBuilds
```

Comment:
I stopped and wondered if we somehow consider the current selection here. Maybe name it `builds`? Or `buildCandidates`?.